manuscript.md

```diff
@@ -36,6 +40,45 @@
 
 ## What is a species interaction?
 
+Interactions between species can be manifested in a multitude of ways e.g. different 
+types [@Jordano2016ChaEco], strengths, or symmetry and are not necessarily restricted 
+between only a pair of species e.g indirect interactions [@Morales-Castilla2015InfBio]. The common thread 
+between these multiple forms of interactions is that *at least* one of the species is 
+affected by the interaction, this can be either positively or negatively [@Morales-Castilla2015InfBio]. Combine 
+this high 'diversity' of interaction types with the sheer number of potential 
+interactions that *could* occur in a community and it is understandable why attempting 
+to sample species interaction empirically is a considerable task. However, inferring 
+interactions through proxies and other frameworks could prove to be promising 
+[@Morales-Castilla2015InfBio], and requires representing (and constraining?) 
+interaction networks in a variety of ways, primarily graphs or matrices.
+
+In a graph, each species is represented as a node and each interaction as an edge
+[@Delmas2018AnaEco, @Pascual2006EcoNet], thus the value/form an edge takes is a 
+representation of the interaction. When using a matrix format we can uses two 
+approaches, namely bipartite networks where we have two interacting species guilds 
+and have one guild be represented by rows and the other by columns e.g. hosts and 
+parasites or we can use unipartite networks where each species *appears* twice - 
+both as a row and column species. For a presence/absence interaction network, the 
+matrix is filled with 1 and 0 representing an interaction and an
+absence of interaction respectfully [@Dunne2006NetStr]. 
```

Comment:
> respectfully

respectively? (but also respectfully, hopefully)

Reply:
We always have to show the interaction our respect!